pom.xml

```diff
@@ -393,34 +393,11 @@
                         </execution>
                     </executions>
                 </plugin>
-                <plugin>
-                    <groupId>io.fabric8</groupId>
-                    <artifactId>fabric8-maven-plugin</artifactId>
-                    <version>4.4.0</version>
-                </plugin>
                 <plugin>
                     <groupId>io.fabric8</groupId>
                     <artifactId>docker-maven-plugin</artifactId>
-                    <version>0.36.0</version>
-                    <extensions>true</extensions>
-                    <executions>
-                        <execution>
-                            <id>before-integration-tests</id>
-                            <phase>pre-integration-test</phase>
-                            <goals>
-                                <goal>stop</goal>
-                                <goal>start</goal>
-                            </goals>
-                        </execution>
-                        <execution>
-                            <id>after-integration-tests</id>
-                            <phase>post-integration-test</phase>
-                            <goals>
-                                <goal>stop</goal>
-                            </goals>
-                        </execution>
-                    </executions>
-                </plugin>
```

Comment:
See previous comment. At minimum these executions should be re-added to this file and remove from the alfresco-transform-core-aio/alfresco-transform-core-aio-boot/pom.xml above.
As mentioned above, it might be possible to move all the config for the `docker-it-setup` profiles to this file.